dashboard/db/migrate/20160818232419_add_url_param_to_stages.rb

```diff
@@ -0,0 +1,35 @@
+class AddUrlParamToStages < ActiveRecord::Migration
+  def up
+    add_column :stages, :url_param, :string
+
+    Stage.reset_column_information
+    # Go through each script, and calculate url_param for lockable/unlockable
+    # stages
+    Script.all.each do |script|
```

Comment:
Do we need to do this in the migration, or will the `rake seed` process re-populate these values?